src/Nexus.js

```diff
@@ -0,0 +1,175 @@
+});
+
+gpii.nexus.constructComponent.handleRequest = function (path, request) {
+    fluid.construct(path, request.req.body);
+    request.events.onSuccess.fire();
+};
+
+fluid.defaults("gpii.nexus.destroyComponent.handler", {
+    gradeNames: ["kettle.request.http"],
+    invokers: {
+        handleRequest: {
+            funcName: "gpii.nexus.destroyComponent.handleRequest",
+            args: ["{request}.req.params.path", "{request}"]
+        }
+    }
+});
+
+gpii.nexus.destroyComponent.handleRequest = function (path, request) {
+    fluid.destroy(path);
+    request.events.onSuccess.fire();
+};
+
+fluid.defaults("gpii.nexus.bindModel.handler", {
+    gradeNames: ["kettle.request.ws"],
+    members: {
+        // We store the targetComponent inside a container so that the
+        // component is isolated from IoC references. This will not be
+        // necessary in the future after upcoming framework changes
+        // are completed.
+        // See https://issues.fluidproject.org/browse/FLUID-4925
+        componentHolder: {
+            targetComponent: null // Will be set at onBindWs
+        },
+        modelPath: null // Will be set at onBindWs
+    },
+    invokers: {
+        targetModelChangeListener: {
+            funcName: "gpii.nexus.bindModel.targetModelChangeListener",
+            args: [
+                "{that}",
+                "{arguments}.0" // value
+            ]
+        }
+    },
+    listeners: {
+        onBindWs: {
+            funcName: "gpii.nexus.bindModel.bindWs",
+            args: [
+                "{that}",
+                "{request}.req.params.componentPath",
+                "{request}.req.params.modelPath",
+                "{that}.targetModelChangeListener"
+            ]
+        },
+        onReceiveMessage: {
+            funcName: "gpii.nexus.bindModel.receiveMessage",
+            args: [
+                "{that}.componentHolder.targetComponent",
+                "{that}.modelPath",
+                "{arguments}.1" // message
+            ]
+        },
+        onDestroy: {
+            "this": "{that}.componentHolder.targetComponent.applier.modelChanged",
+            method: "removeListener",
+            args: ["{that}.targetModelChangeListener"]
+        }
+    }
+});
+
+// TODO: Support both string and array paths
+// TODO: Move gpii.nexus.componentForPath to infusion FluidIoC.js "BEGIN NEXUS METHODS"
+gpii.nexus.componentForPath = function (path) {
+    return fluid.globalInstantiator.pathToComponent[path];
+};
+
+gpii.nexus.bindModel.bindWs = function (handler, componentPath, modelPath, modelChangeListener) {
+    handler.componentHolder.targetComponent = gpii.nexus.componentForPath(componentPath);
+    // TODO: Note that applier.modelchanged.addListener is different from https://wiki.gpii.net/w/Nexus_API
+    //       Which says applier.addModelListener
+    handler.componentHolder.targetComponent.applier.modelChanged.addListener(modelPath, modelChangeListener); // TODO: namespace?
```

Comment:
Namespaces will be properly supported once https://github.com/fluid-project/infusion/pull/671 is merged into the core framework